kernel/aster-nix/src/fs/utils/page_cache.rs

```diff
@@ -341,8 +341,9 @@ impl PageCacheManager {
     // Discard pages without writing them back to disk.
     pub fn discard_range(&self, range: Range<usize>) {
         let page_idx_range = get_page_idx_range(&range);
+        let mut pages = self.pages.lock();
         for idx in page_idx_range {
-            self.pages.lock().pop(&idx);
+            pages.pop(&idx);
```

Comment:
Next time, do not piggyback changes that are irrelevant to the description of PRs and commits. This PR is supposed to do renaming, only.